src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx

```diff
@@ -23,14 +13,21 @@ import {
 import { TypeRegistry } from "@polkadot/types";
 import { Tokens } from "@chainsafe/web3-context/dist/context/tokensReducer";
 import { BigNumber as BN } from "bignumber.js";
-import { UnsubscribePromise, VoidFn } from "@polkadot/api/types";
+import { VoidFn } from "@polkadot/api/types";
 import { utils } from "ethers";
 import {
   SubstrateBridgeConfig,
-  getСhainTransferFallbackConfig,
+  chainbridgeConfig,
 } from "../../chainbridgeConfig";
 import { toFixedWithoutRounding } from "../../Utils/Helpers";
-import { Fallback } from "../../Utils/Fallback";
+import AnalyticsService from "../../Services/Analytics";
+const analytics = new AnalyticsService({
```

Comment:
Can it be initialized once?

Reply:
Theoretically yes,  we can try to create a separate  <AnalyticsProvider> or put it to the existing <NetworkManagerProvider>. What is more preferable?